sections/salinity_mission_execution.md

```diff
@@ -4,7 +4,7 @@
 This section covers the activities of those deploying and recovering the gliders in the field in addition to best practices for glider pilots.
 
 (deploy)=
-## Deployment
+## Deployment and recovery
```

Comment:
As we try to keep the deployment life cycle as the main structure we should keep the old title her but link this section in the post-deployment section related to #29

Reply:
You can see how we did this in the oxygen SOP in section 7.4 "Field calibration" 
https://oceangliderscommunity.github.io/Oxygen_SOP/sections/oxygen_post_recovery.html